src/dom-to-image-more.js

```diff
@@ -89,6 +89,7 @@
 
         function clearCache(result) {
             domtoimage.impl.urlCache = [];
+            removeSandbox();
```

Comment:
I'm going to do this change in a separate PR just for neatness.

Reply:
... and released that change in both v2.14.1 and v3.0.2-rc.0

Reply:
... and WOO! as of this weekend, we're getting picked up by CDNJS :) https://cdnjs.com/libraries/dom-to-image-more